app/views/vacancies/_filters.html.haml

```diff
@@ -1,48 +1,17 @@
 - content_for :advanced_filters do
-  .govuk-form-group
+  .govuk-form-group.app-site-search
+    = label_tag 'job_title', 'Keyword', class: 'govuk-label'
+    = search_field_tag :job_title, job_title, class: 'govuk-input', placeholder: t('jobs.filters.job_title_hint')
```

Comment:
form field to submit should be :keyword not :job_title

Reply:
gotcha, my misunderstanding with haml

Reply:
this will be addressed when i integrate updated rspec tests with this feature branch